html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue

```diff
@@ -51,6 +51,10 @@
               <icon class="position-absolute mt-1" name="project-diagram"></icon>
               <span class="ml-4">{{ $t('Apply Bypass VLAN') }}</span>
             </b-dropdown-item>
+            <b-dropdown-item @click="applyBulkDelete()">
+              <icon class="position-absolute mt-1" name="trash-alt"></icon>
+              <span class="ml-4">{{ $t('Delete') }}</span>
```

Comment:
I'm really not sure about allowing the ability to bulk delete.

This has the potential to cause a sh*tload of problems.

I'm fine with leaving the API call there and the admin plumbing but I'd remove the option for now